docs/getting-started/bpfctl-guide.md

```diff
@@ -110,17 +110,25 @@ Options:
   -g, --global <GLOBAL>...
           Optional: Global variables to be set when program is loaded.
           Format: <NAME>=<Hex Value>
-
+          
           This is a very low level primitive. The caller is responsible for formatting
           the byte string appropriately considering such things as size, endianness,
           alignment and packing of data structures.
 
+      --map-owner-uuid <MAP_OWNER_UUID>
+          Optional: Uuid of loaded eBPF program this eBPF program will share a map with.
+          Only used when multiple eBPF programs need to share a map. If a map is being
+          shared with another eBPF program, the eBPF program that created the map can not
```

Comment:
This needs to some further clarification. You CAN remove the first program before the second one, but you'll put `bpfd` in a bad state when the next XDP/TC program is loaded since the pin path will no longer exist.

Reply:
I added a check that prevents it from being unloaded. I can remove this if needed.
https://github.com/bpfd-dev/bpfd/blob/main/bpfd/src/bpf.rs#L1230